spec/api/payment/void_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Afterpay::API::Payment::Void do
+  describe '#call' do
+    let(:token) { 'some_token' }
+    let(:order_id) { '12345' }
+    let(:payment) { build(:payment, token: token) }
+
+    context 'with proper params' do
+      let(:raw_response) { JSON.generate(paymentState: 'VOIDED') }
+
+      before(:each) do
+        stub_request(:post, %r{api.us.afterpay.com/v2/payments/#{order_id}/void})
+          .to_return(
+            status: 201,
+            body: raw_response,
+            headers: { 'Content-Type' => 'application/json' }
+          )
+      end
+
+      it 'returns hashie mash' do
+        expect(described_class.call(payment: payment, order_id: order_id, amount: "12.11").body).to be_instance_of Hashie::Mash
+      end
+
+      it 'returns afterpay order id' do
+        expect(described_class.call(payment: payment, order_id: order_id, amount: "12.11").body).to include(:paymentState)
```

Comment:
the amount should be a `Afterpay::Components::Money` and should be included in the payment object.

Reply:
Added the amount as a money object like requested!